### ISSUE TITLE
[bug]Vssue can not work

### ISSUE THREAD

**Issue by chavesgu** [Yubisaki/vuepress-theme-yubisaki#32]
i found that `index.js` in your theme dir,
```js
module.exports = ({ comments = {} }) => ({
  layoutDir: 'layouts',
  plugins: [
    ['@vssue/vuepress-plugin-vssue', Object.assign({
      platform: 'github',
    }, comments)],
  ]
})
```
i think it should be `comment`, not `comments`

**Comment by Bloss** [comment]
It works fine on my blog.
No matter what the variable is, comment or comments, it won't cause vssue crash. 
Any repo to reproduce?

**Comment by chavesgu** [comment]
> It works fine on my blog.
> No matter what the variable is, comment or comments, it won't cause vssue crash.
> Any repo to reproduce?

Comments on your site， now it's unable to log in, when i click login   , cause 404

**Comment by Bloss** [comment]
fixed in c8e2892
